claimManagement/src/main/java/org/openimis/imisclaims/ToRestApi.java

```diff
@@ -53,7 +52,6 @@ public HttpResponse postToRestApi(final JSONObject object, final String function
 
     //Post with Token
     public HttpResponse postToRestApiToken(final JSONObject object, final String functionName) {
-        setProperUri(functionName);
```

Comment:
You removed this function call. Is this to manage the 2 REST APIs? Should this be removed only after REST APIs integration?

Reply:
I fixed the setProperUri function.